streamer/pulp/streamer/adapters.py

```diff
@@ -123,19 +123,23 @@ def proxy_manager_for(self, proxy, **proxy_kwargs):
 
         return self.proxy_manager[proxy]
 
-    def _update_poolmanager_ssl_kw(self, verify, cert):
+    def _update_poolmanager_ssl_kw(self, pool_manager, verify, cert):
         """Update the :class:`PoolManager <urllib3.poolmanager.PoolManager>`
         connection_pool_kw with the necessary SSL configuration. This method
         should not be called from user code, and is only exposed for use when
         subclassing the :class:`HTTPAdapter <requests.adapters.HTTPAdapter>`.
 
+        :param pool_manager: A pool manager to be updated.
+        :type  pool_manager: urllib3.poolmanager.PoolManager
         :param verify: Whether we should actually verify the certificate;
                        optionally a path to a CA certificate bundle or
                        directory of CA certificates.
+        :type  verify: str
```

Comment:
I think you're right so I changed to a `str` but the name and code below are confusing:

```
        if verify:

            cert_loc = None

            # Allow self-specified cert location.
            if verify is not True:
                cert_loc = verify
```